src/irmin/tree.ml

```diff
@@ -423,8 +424,13 @@ module Make (P : Private.S) = struct
     (* export t to the given repo and clear the cache *)
     let export ?clear:(c = true) repo t k =
       let hash = t.info.hash in
-      if c then clear t;
+      if c then (
+        t.info.value <- None;
+        t.info.map <- None;
+        t.info.hash <- None;
+        t.info.findv_cache <- None);
```

Comment:
we probably don't need this anymore with https://github.com/mirage/irmin/pull/1515

Reply:
Fixed